src/tick-trade/TickTradeCard.js

```diff
@@ -66,6 +67,7 @@ export default class TickTradeCard extends Component {
                 />
                 <BuyButton
                     askPrice={askPriceFromProposal(trade.proposal)}
+                    isAuthorized={isAuthorized}
```

Comment:
Too much 'leak' of information. We don't want each component to be aware of authorization. Disabled should be used in this case.